integ/components/deadline/deadline_01_repository/bin/deadline_01_repository.ts

```diff
@@ -16,20 +16,25 @@ const env = {
 const integStackTag = process.env.INTEG_STACK_TAG!.toString();
 
 const componentTier = new Stack(app, 'RFDKInteg-DL-ComponentTier' + integStackTag, {env});
-const storage1 = new StorageStruct(componentTier, 'StorageStruct1', {
-  integStackTag,
-  provideDocdbEfs: false,
-  useMongoDB: false,
-});
-const storage2 = new StorageStruct(componentTier, 'StorageStruct2', {
-  integStackTag,
-  provideDocdbEfs: true,
-  useMongoDB: false,
-});
-const storage3 = new StorageStruct(componentTier, 'StorageStruct3', {
-  integStackTag,
-  provideDocdbEfs: false,
-  useMongoDB: true,
-});
 
-new TestingTier(app, 'RFDKInteg-DL-TestingTier' + integStackTag, { env, integStackTag, structs: [storage1, storage2, storage3] });
+const structs: Array<StorageStruct> = [];
```

Comment:
I think we can initialize this array in-play, instead of using `push`:
```suggestion
const structs: Array<StorageStruct> = [
  new StorageStruct(componentTier, 'StorageStruct1', {
    integStackTag,
    provideDocdbEfs: false,
    useMongoDB: false,
  }),
  new StorageStruct(componentTier, 'StorageStruct2', {
    integStackTag,
    provideDocdbEfs: true,
    useMongoDB: false,
  }),
  new StorageStruct(componentTier, 'StorageStruct3', {
    integStackTag,
    provideDocdbEfs: false,
    useMongoDB: true,
  }),
];
```

Reply:
Agreed. Change made.